src/index.ts

```diff
@@ -1,56 +1,67 @@
 import withApiData from './withApiData';
-import { configureApiData } from './actions/configureApiData';
+import { configure } from './actions/configure';
 import { afterRehydrate } from './actions/afterRehydrate';
-import { purgeAllApiData } from './actions/purgeAllApiData';
-import { getApiDataRequest } from './selectors/getApiDataRequest';
+import { purgeAll } from './actions/purgeAll';
+import { getRequest } from './selectors/getRequest';
 import { getResultData } from './selectors/getResultData';
-import { invalidateApiDataRequest } from './actions/invalidateApiDataRequest';
-import { performApiRequest, useRequestHandler } from './actions/performApiDataRequest';
+import { invalidateRequest } from './actions/invalidateRequest';
+import { performApiRequest, useRequestHandler } from './actions/performRequest';
 import { getEntity } from './selectors/getEntity';
 import reducer from './reducer';
-import { ApiDataState } from './reducer';
+import { State } from './reducer';
 import useApiData from './useApiData';
 import useActions from './useActions';
 import {
     NetworkStatus,
     NormalizeResult,
     NormalizedData,
     EndpointParams,
-    ApiDataRequest,
-    ApiDataGlobalConfig,
+    Request,
+    GlobalConfig,
     Method,
-    ApiDataEndpointConfig,
-    ApiDataConfigBeforeProps,
-    ApiDataConfigAfterProps,
-    ApiDataBinding,
+    EndpointConfig,
+    ConfigBeforeProps,
+    ConfigAfterProps,
+    Binding,
     Actions,
 } from './types';
 
 export {
     withApiData,
-    configureApiData,
+    configure,
     performApiRequest,
-    getApiDataRequest,
+    getRequest,
     getResultData,
     getEntity,
-    invalidateApiDataRequest,
+    invalidateRequest,
     afterRehydrate,
-    purgeAllApiData,
+    purgeAll,
     reducer,
     useRequestHandler,
-    ApiDataState,
+    State,
     useApiData,
     useActions,
     NetworkStatus,
     NormalizeResult,
     NormalizedData,
     EndpointParams,
-    ApiDataRequest,
-    ApiDataGlobalConfig,
+    Request,
+    GlobalConfig,
     Method,
-    ApiDataEndpointConfig,
-    ApiDataConfigBeforeProps,
-    ApiDataConfigAfterProps,
-    ApiDataBinding,
+    EndpointConfig,
+    ConfigBeforeProps,
+    ConfigAfterProps,
+    Binding,
     Actions,
 };
+
+export const configureApiData = configure;
+export const getApiDataRequest = getRequest;
+export const purgeAllApiData = purgeAll;
+export type ApiDataState = State;
+export type ApiDataRequest = Request;
```

Comment:
"Request" conflicteert met de Request van de fetch API in de global namespace. Ik weet niet of dit een probleem geeft. Misschien beter renamen naar "DataRequest" (of als je een betere suggestie hebt..)